modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -530,6 +530,8 @@ TEST_P(Test_Darknet_layers, avgpool_softmax)
 
 TEST_P(Test_Darknet_layers, region)
 {
+     if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NGRAPH);
```

Comment:
Why we skip it?

Reply:
because in this test case check `normAsssert `without NMS sorting

Reply:
Got it, thanks!